### PR TITLE
app-emulation/crun: version bump to 0.10.4

### PR DESCRIPTION
**Pull request by dmolik**
Package-Manager: Portage-2.3.78, Repoman-2.3.17
Signed-off-by: Dan Molik <dan@danmolik.com>
Tested-by: Dan Molik <dan@danmolik.com>

### CODE REVIEW

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit eutils python-any-r1
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: Hmm, I didnt see eutils being used in here?

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+DESCRIPTION="A fast and low-memory footprint OCI Container Runtime fully written in C"
+HOMEPAGE="https://github.com/containers/crun"
+SRC_URI="https://github.com/containers/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: https://github.com/containers/crun/blob/master/COPYING.libcrun
What's this? Could you verify the licenses altogether?

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="bpf +caps doc seccomp systemd static-libs"
+
+DEPEND="
+	dev-libs/yajl
+	sys-libs/libseccomp
+	caps? ( sys-libs/libcap )
+	seccomp? ( sys-libs/libseccomp )
+	systemd? ( sys-apps/systemd )
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: If it links to systemd (I dont know if it does or doesnt) you'll want to add a subslot binder here, `:=`.

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=dev-lang/python-3.5.0
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: `${PYTHON_DEPS}` might be what you're looking for in here.

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+		$(use_enable caps) \
+		$(use_enable seccomp) \
+		$(use_enable systemd) \
+		$(usex static-libs '--enabled-shared  --enabled-static' '--enable-shared --disable-static' '' '')
+}
+
+src_compile() {
+	pushd libocispec || die
+	emake
+	popd
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: `|| die`

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+	emake crun
+	if use doc ; then
+		emake crun.1
+	fi
+}
+
+src_install() {
+	pushd libocispec || die
+	emake "DESTDIR=$D" install-exec
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: Please add curly brackets here, `${D}` and everywhere below.

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+	popd
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: `|| die`

File: app-emulation/crun/crun-0.10.4.ebuild [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=dev-lang/python-3.5.0
+	doc? ( dev-go/go-md2man )
+"
+
+DOCS=( README.md )
+
+src_configure() {
+	econf \
```
Comment (juippis) [inline app-emulation/crun/crun-0.10.4.ebuild]: This is not wrong and you can leave it if you want, but just letting you know there's a more foolproof and more prettier array form usable.

**Comment by dmolik** [comment]
Thank you for the feedback, I really appreciate, not many people are willing to spend the time. I'll try and get changes in later today

**Comment by dmolik** [comment]
@juippis I've incorporated your feedback, but given a recent security concern I'm not going to publish 0.10.4. Instead, I've opened a new PR with 0.10.5 ( #13597 )

**Pull request closed without merging**